bundle/jsky.app.ot/src/main/scala/jsky/app/ot/viewer/open/OpenDialog.scala

```diff
@@ -110,21 +112,27 @@ class OpenDialog private(db: IDBDatabaseService, auth: KeyChain, vcs: VcsRegistr
     override def apply(t: Option[(Peer, Key)]): Unit = Contents.ProgTable.refresh(full = true)
   })
 
-  var selection: Option[ISPProgram] = None
+  var selection: List[ISPProgram] = Nil
   val filter = new DBProgramChooserFilter(DBProgramChooserFilter.Mode.localAndRemote)
 
+  val MaxM = 50
+  val MaxK = MaxM * 1000
```

Comment:
Why 1000 instead of 1024, out of curiosity?

Reply:
For me, it was continuity with what was there before.  It does seem like it should be 1024.  In the end it doesn't matter too much because we are using the size of the serialized file to give us an idea of the size of how much memory the program takes when loaded.  That's pretty sketchy anyway and the 50M maximum is only a guess for when the OT might start getting bogged down.